pkg/ccl/streamingccl/streamingest/datadriven_test.go

```diff
@@ -425,6 +447,16 @@ func (d *datadrivenTestState) execAs(t *testing.T, as, query string) {
 	}
 }
 
+func handleRegex(t *testing.T, err error, regexError string) error {
+	matched, matchStringErr := regexp.MatchString(regexError, err.Error())
+	require.NoError(t, matchStringErr)
+	if matched {
+		return nil
+	} else {
+		return errors.Wrapf(err, "error does not match regex error %s", regexError)
```

Comment:
I think there is another gotcha potential here. If the query _doesn't_ return an error and err is nil then `errors.Wrapf(nil,...)` also returns nil.  In this case, I think you'll fail before that with a NPE at line 451.

Reply:
oof. good call. apologies for the general brain fog on this pr.